src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/InputIncorrectSummaryJavaDoc.java

```diff
@@ -5,17 +5,17 @@
  * is a simple Javadoc. Some javadoc.
  */
 class InputIncorrectSummaryJavaDoc {
-    
+
     /**
-     * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}
+     * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}.
```

Comment:
don't fix 'bad' javadocs in a 'bad' input file.
We expect violations here to make sure your logic is working correctly.